node/configuration.yaml

```diff
@@ -305,6 +307,8 @@ mainnet_base: &mainnet_base
     blockRetrievalQueueSize: 100
     propagationQueueSize: 100
     pendingTxResubmissionPeriod: 7 # seconds
+    walletProductionApi: false
```

Comment:
Shouldn't it be `true` by default in mainnet config? I think it's safer. Even if it's temporary. I. e. let's be explicit that we want to enable some dangerous endpoints.